pkg/v1/tkg/client/common.go

```diff
@@ -140,6 +201,24 @@ func (c *TkgClient) validateDockerPrerequisites() error {
 	if !isDockerDaemonRunning {
 		return errors.New("docker daemon is not running, Please make sure Docker daemon is up and running")
 	}
+	// validate docker allocated CPU and memory against recommended minimums
+	if validateDockerResources {
```

Comment:
I'm just curious. Why are these checks optional? Why are they not part of the prerequisites?

Reply:
@saji-pivotal The original checks for docker daemon running serve two purposes. First, we are validating that docker is running prior to executing the CLI create command for any provider. Second, this is repurposed for the UI to check if the daemon is running at the start of the docker workflow...the step referenced in the screen shots. So what we have to do is determine what is the source of triggering this validation to occur. If it is coming from the UI to validate that the daemon is running, then we also want to confirm the user has the appropriate resource minimums set. 

We would want to bypass those memory/cpu resource checks when the create command is run for AWS, vSphere, Azure.

Reply:
Agreed, much better approach. Thank you for the suggestion.